ConsoleApp/Validator.cs

```diff
@@ -0,0 +1,25 @@
+namespace Bank
+{
+    public static class Validator
+    {
+        public static bool IsPositive(decimal amount)
+        {
+            if (amount <= 0)
+            {
+                Console.WriteLine("Error! Value could not be negative or equals zero");
```

Comment:
do not create methods that add to their main purpose write something to the console. if I'll use your classes inside other application, you can broke my user interface with your text